appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/RecoveryManager.java

```diff
@@ -219,7 +219,7 @@ static void initialise() {
             try {
                 recoveryInProgress.post(); // BUGFIX (Ram Jeyaraman)
                 resyncComplete(false, false);
-            } catch (Throwable exc) {}
+            } catch (Throwable exc) {exc.printStackTrace();}
```

Comment:
this is nasty. it's slightly less nasty than silently swallowing the exception, but still nasty:)